tests/reporter/kubernetes.go

```diff
@@ -606,7 +606,7 @@ func (r *KubernetesReporter) logJournal(virtCli kubecli.KubevirtClient, logsdir
 		commands := []string{
 			virt_chroot.GetChrootBinaryPath(),
 			"--mount",
-			virt_chroot.GetChrootMountNamespace(),
+			virt_chroot.GetChrootNSMountPath(),
```

Comment:
Same as above

Reply:
@xpivarc this GetChrootNSMountPath() is:
```
const (
	mountNamespace = "/proc/1/ns/mnt"
)

func GetChrootNSMountPath() string {
	return mountNamespace
}
```
is basically "/proc/1/ns/mnt", regardless of PID.
or do i miss something?

Reply:
My bad, thank you!